samples/Chat/src/app/ChatHeader.tsx

```diff
@@ -26,6 +30,14 @@ export const ChatHeader = (props: ChatHeaderProps): JSX.Element => {
 
   return (
     <Stack horizontal={true} verticalAlign={'center'} horizontalAlign="end" className={chatHeaderContainerStyle}>
+      {/** FIXME: Similar to #1235 */}
+      <div className={paneButtonContainerStyle}>
+        <IconButton
+          onRenderIcon={() => (props.isParticipantsDisplayed ? <People20Filled /> : <People20Regular />)}
+          className={mergeStyles({ color: theme.palette.neutralPrimaryAlt })}
+          onClick={() => props.setHideParticipants(props.isParticipantsDisplayed)}
```

Comment:
missing ariaLabel prop on this one. As it's an icon with no label, screen reader will not know what it is apart being a button, so we need to add an ariaLabel string like ' Participant list' or 'participant pane'

Reply:
I've added a FIXME above, will upload a follow up PR. I don't want to make new functional changes in this PR. This is as much as possible:

`git diff --binary main..<feature-branch-sha> | git patch` :)

Reply:
https://github.com/Azure/communication-ui-library/pull/1261 up for this.